src/index.ts

```diff
@@ -76,6 +76,13 @@ function pluginTerminal(options: Options = {}) {
                 config.logger.info(`» ${table(obj, indent)}`)
                 break
               }
+              case 'log': {
+                let obj = JSON.parse(message)
+                if (Array.isArray(obj))
+                  obj = obj.length === 1 ? JSON.stringify(obj[0], null, 2) : obj.toString()
+                config.logger.info(colors.log(`» ${obj}`))
+                break
+              }
```

Comment:
I think it is better to avoid doing the processing in the plugin, and directly format the message in the client. Let's directly send a single string to the server if multiple objects are passed

Reply:
Yes, thought the same, but as table was working on that way I wanted to follow the same flow, will make the changes.